sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/DatabasePersistence.java

```diff
@@ -109,6 +133,9 @@ public void onError(String operation, RuntimeException e) {
                         AppCenterLog.error(LOG_TAG, "Cannot complete an operation (" + operation + ")", e);
                     }
                 });
+        mLargePayloadDirectory = new File(Constants.FILES_PATH + PAYLOAD_LARGE_DIRECTORY);
+        //noinspection ResultOfMethodCallIgnored we handle errors at read/write time for each file.
```

Comment:
Didn't we have a line break before the line?

Reply:
adding